website/docs/gitops-templates/templates.mdx

```diff
@@ -0,0 +1,231 @@
+  templates.weave.works/kustomizations-enabled: true
+  templates.weave.works/credentials-enabled: true
+```
+
+## Default profile values
+
+Default and required profiles can be added via the template annotation `capi.weave.works/profile-INDEX` so that if the profiles section has been enabled on a template you can choose those profiles for the template.
+
+The annotation is added as the following:
+```
+annotations:
+    capi.weave.works/profile-0: '{"name": "NAME", "version": "VERSION", "editable": EDITABLE, "namespace": "NAMESPACE"}'
+```
+Where
+  - `name` -  is the name of the profile in the default profiles repository
+  - `version` -  (optional) will choose the default version
+  - `namespace` -  (optional) is the default target namespace for the profile
+  - `editable` -  (optional, default=false), allow the user to de-select this profile, making it a default instead of a requirement.
+
+## Rendering Templates
+
+Declare the render type indicating the templating language to be used to render the template by setting `spec.renderType`.
+
+Supported templating languages:
+  - **envsubst (default)**
+    envsubst which is short for environment substitution uses [envsubst](https://github.com/a8m/envsubst) for rendering, where `${CLUSTER_NAME}` style syntax can be used. It is the same templating format that is used by [clusterctl](https://cluster-api.sigs.k8s.io/clusterctl/overview.html).
+
+    #### Supported Functions
+
+    | __Expression__                | __Meaning__                                                     |
+    | -----------------             | --------------                                                  |
+    | `${var}`                      | Value of `$var`
+    | `${#var}`                     | String length of `$var`
+    | `${var^}`                     | Uppercase first character of `$var`
+    | `${var^^}`                    | Uppercase all characters in `$var`
+    | `${var,}`                     | Lowercase first character of `$var`
+    | `${var,,}`                    | Lowercase all characters in `$var`
+    | `${var:n}`                    | Offset `$var` `n` characters from start
+    | `${var:n:len}`                | Offset `$var` `n` characters with max length of `len`
+    | `${var#pattern}`              | Strip shortest `pattern` match from start
+    | `${var##pattern}`             | Strip longest `pattern` match from start
+    | `${var%pattern}`              | Strip shortest `pattern` match from end
+    | `${var%%pattern}`             | Strip longest `pattern` match from end
+    | `${var-default`               | If `$var` is not set, evaluate expression as `$default`
+    | `${var:-default`              | If `$var` is not set or is empty, evaluate expression as `$default`
+    | `${var=default`               | If `$var` is not set, evaluate expression as `$default`
+    | `${var:=default`              | If `$var` is not set or is empty, evaluate expression as `$default`
```

Comment:
From https://github.com/a8m/envsubst, I think these should still have the closing `}`